### PR TITLE
Experiment

### PR DESCRIPTION
**Pull request by arnaldoaparicio**
# Description
Google OAuth2 is now fully functional on Heroku.

Please check the type of change your PR introduces:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

 

 

### Type of change

Please check the relevant option.

- [x] Non-Breaking Change
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Big Breaking change (this breaks everything and I am sorry)

# How Has This Been Tested?

Please check the option(s) that best describe your tests. 

- [ ] New spec files were added
- [ ] Tests were added to existing spec files
- [ ] Some tests were changed
- [ ] No tests were changed
- [ ] All tests were changed (please explain what in God's name happened)

### Tests Description

- Please add further details of your tests here.

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have removed all unused code or commented why I left it
- [ ] I have deployed my changes to a test-heroku and they work
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

**Comment by Malllll12** [comment]
Arnaldo does this work on Heroku???

**Comment by arnaldoaparicio** [comment]
> Arnaldo does this work on Heroku???

Yes, although not on the Heroku links we had initially. They're working on my personal Heroku links.

**Review by Sierra-T-9598 — COMMENTED** [review]
Might be worth getting these last few tests passing before we merge into main. Are they passing RSpec on your end Arnaldo?

**Comment by arnaldoaparicio** [comment]
> Might be worth getting these last few tests passing before we merge into main. Are they passing RSpec on your end Arnaldo?

Nah they're not passing. I'll look into it further.

**Review by Malllll12 — APPROVED** [review]

**Comment by Sierra-T-9598** [comment]
Awesome work! Sanitize method, very cool!

**Review by Sierra-T-9598 — APPROVED** [review]

File: spec/features/books/index_spec.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
     scenario 'I see a flash message if query returns no results' do
       expect(page).to have_button('Search')
-
+      # binding.pry
       fill_in :search, with: "jh21g31yiu2g3"
```
Comment (jbreit88) [inline spec/features/books/index_spec.rb:19]: looks like a pry is left in the books/index_spec

**Review by jbreit88 — COMMENTED** [review]
Just one pry I saw hidden in there. Beyond that everything looks good to me.

**Pull request merged**